audio/whisper/Portfile

```
@@ -0,0 +1,105 @@
+#-*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup   github  1.0
+
+name                whisper
+categories          audio
+github.setup        ggerganov whisper.cpp v1.6.2
```

`github.setup`'s format is `<author> <project name> <version> <version_prefix> <version_suffix>`, so this would instead be:

```suggestion
github.setup        ggerganov whisper.cpp 1.6.2 v
```

---

Thanks, I'll try this again. I initially started this way, but I was running into some kind of trouble with downloading the .tar.gz release file.

---

You may need to add `github.tarballs_from archive`

---

The checksums will need to be updated if you switch to using Github's archive source